roles/drain_nodes/tasks/main.yml

```diff
@@ -6,10 +6,14 @@
     {% if delete_local_data|bool %}
     --delete-local-data
     {% endif %}
+    {% if force_drain|bool %}
```

Comment:
What do you think about 'drain_force' ? With "force_drain", I thought an option for forcing drain to occur, not forcing the drain itself.

Reply:
Yep, it's better, fixed :)